src/routes.ts

```diff
@@ -87,6 +88,22 @@ export default [
       ]
     }
   },
+  {
+    path: '/notice-of-withdrawal',
+    name: Routes.NOTICE_OF_WITHDRAWAL,
```

Comment:
That looks good.

What was it before?

Reply:
It was path: '/notice-of-withdrawal/:filingToBeWithdrawn',

BC0883739/notice-of-withdrawal/153004?filingId=0

Reply:
Oh, yeah, that is different. It was a dynamic route instead of a route query parameter. You were extracting it from params but I think that's unusual.

FYI, there's one more way this could be done, and that's to use a colon for the parameter in the route spec, and then have the router convert the parameter to a prop for the component to use. Example: https://github.com/bcgov/sbc-auth/blob/d56df39c91c515dc0224bc72b0ec925c419de5c1/auth-web/src/routes/router.ts#L235

Reply:
Don't change your code. Resolving.